src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -29,6 +29,10 @@ const timecardService = require('../../../api/services/timecardService');
 const mockCreateTimeEntry = jest.spyOn(timecardService, 'createTimeEntry');
 const mockUpdateTimeEntry = jest.spyOn(timecardService, 'updateTimeEntry');
 
+beforeEach(() => {
```

Comment:
I'd just double-check this is actually needed. Only because it will run before every test and I can see there's a lot of tests

Reply:
That makes complete sense - just wanted to check.